core/storage/changes_trie/changes_tracker.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef KAGOME_STORAGE_CHANGES_TRIE_CHANGES_TRACKER
+#define KAGOME_STORAGE_CHANGES_TRIE_CHANGES_TRACKER
+
+#include "common/buffer.hpp"
+#include "primitives/extrinsic.hpp"
+#include "storage/changes_trie/changes_trie_builder.hpp"
+#include "storage/changes_trie/changes_trie_config.hpp"
+
+namespace kagome::storage::changes_trie {
+
+  /**
+   * Stores the information about changes to the storage in accordance with the
+   * supplied config. Used to build the changes trie.
+   * onChange() must be called every time a persistent change to the node
+   * storage is made.
+   */
+  class ChangesTracker {
+   public:
+    typedef std::function<outcome::result<common::Buffer>()>
```

Comment:
be modern, use using:)

Reply:
Yeah, just when started typing, for some reason started with the function type first and then shrugged and made a typedef. Was kinda sleepy.